package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "scripts": {
-    "gr": "node local_garment/node_modules/.bin/garment",
+    "gr": "node local_garment/node_modules/@garment/cli/lib/cli.js",
```

Comment:
Using this path doesn't work on Windows so I changed it to point to the file that would be invoked by those bin files anyway.  @beshanoe do you see any problem with this?

Reply:
> Using this path doesn't work on Windows so I changed it to point to the file that would be invoked by those bin files anyway.  @beshanoe do you see any problem with this?

I don't, you're the only person now who can make sure garment works on windows😄 so any catches like this are helpful